go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/containernetworking/plugins v0.8.7
 	github.com/contiv/libOpenflow v0.0.0-20210312221048-1d504242120d
 	github.com/contiv/ofnet v0.0.0-00010101000000-000000000000
-	github.com/coreos/go-iptables v0.4.5
+	github.com/coreos/go-iptables v0.6.0
```

Comment:
this update is needed?

Reply:
Yes, I remember that with old `go-iptables`, some methods are missing.

IIRC, it is `ChainExists()`, firstly introduced by https://github.com/antrea-io/antrea/pull/2458. @tnqn it is correct?

Reply:
yes